content/news/033/index.md

```diff
@@ -33,12 +33,21 @@ Feel free to send PRs about your own projects!
 
 - [Rust GameDev Meetup](#rust-gamedev-meetup)
 - [Game Updates](#game-updates)
-- [Learning Material Updates](#learning-material-updates)
+  - [Way of Rhea](#way-of-rhea)
+  - [BITGUN](#bitgun)
+  - [Veloren](#veloren)
 - [Engine Updates](#engine-updates)
+  - [Bevy v0.7](#bevy-v07)
+  - [Dims](#dims)
+  - [[Eldiron]](#eldiron)
+  - [Hotham](#hotham)
+- [Learning Material Updates](#learning-material-updates)
 - [Tooling Updates](#tooling-updates)
+  - [Vismut 0.5](#vismut-05)
 - [Library Updates](#library-updates)
-- [Other News](#other-news)
+  - [Notan v0.3.0](#notan-v030)
 - [Popular Workgroup Issues in Github](#popular-workgroup-issues-in-github)
+- [Other News](#other-news)
```

Comment:
Ooh this is interesting, do you have something that auto-generated this? We normally don't include the items themself in the list :+1:

Reply:
I have.. no idea! 😆